src/tcp_wrap.h

```diff
@@ -64,6 +65,10 @@ class TCPWrap : public ConnectionWrap<TCPWrap, uv_tcp_t> {
  private:
   typedef uv_tcp_t HandleType;
 
+  bool noDelay;
+  bool keepAlive;
+  unsigned int keepAliveDelay;
```

Comment:
**If** this PR adds new fields, then they should be using `lower_snake_case_;` as elsewhere in the code

Reply:
Sorry, I missed that. No worries, I'll fix once we agree on the final approach.